pytential/symbolic/execution.py

```diff
@@ -705,6 +728,15 @@ def __init__(self, places, auto_where=None):
 
         # {{{ validate
 
+        # check auto_where
+        if auto_source.geometry not in self.places:
+            raise ValueError("'auto_where' source geometry is not in the "
+                f"collection: '{auto_source.geometry}'")
+
+        if auto_target.geometry not in self.places:
+            raise ValueError("'auto_where' target geometry is not in the "
+                f"collection: '{auto_target.geometry}'")
```

Comment:
Is this too strict? It caused a few errors in the tests.

My reasoning was that this would fail later (if autotagging in `bind`) in a more cryptic way.

Reply:
I agree that this check is a good idea.